lib/queue.js

```diff
@@ -551,6 +551,13 @@ Queue.prototype.processStalledJob = function(job){
           if(!isMember){
             return _this.processJob(job);
           }
+
+          return job.releaseLock(_this.token);
+        }).catch(function(err) {
```

Comment:
The benefits of using "disposers" are not entirely clear to me though. I trust to your evaluation of the alternatives for error handling here.